frontend/src/AlertConfiguration/AlertConfigurationTests/AlertConfigurationSetupTest.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```

Comment:
I don't think you need this file by the way if you want to delete it I think it should be okay!

Reply:
You can ignore this file